pyrenew/metaclass.py

```diff
@@ -125,6 +122,38 @@ def _assert_sample_and_rtype(
     return None
 
 
+def compute_delay_ascertained_incidence(
```

Comment:
This appears to be out of date.

Reply:
removed in https://github.com/CDCgov/PyRenew/pull/412/commits/e1b888af1989c3d605a79507522ecaa583640958